src/server.c

```diff
@@ -4627,6 +4627,8 @@ sds genRedisInfoString(const char *section) {
             "total_writes_processed:%lld\r\n"
             "io_threaded_reads_processed:%lld\r\n"
             "io_threaded_writes_processed:%lld\r\n",
+            (long)current_server_time.tv_sec,
+            (long)current_server_time.tv_usec,
```

Comment:
since this is printed in one line i think i rather these to be in one line too.
one line of arguments for each one line of output.
```suggestion
            (long)current_server_time.tv_sec, (long)current_server_time.tv_usec,
```